docker/run_both.sh

```diff
@@ -8,21 +8,22 @@
 # Before we do anything, we need to add the proper env vars for Intel MKL, etc.
```

Comment:
what is both?

Reply:
Not clue what any of these do.  Both might be sender + agora